drivers/flash/Kconfig

```diff
@@ -50,8 +50,6 @@ source "drivers/flash/Kconfig.nios2_qspi"
 
 source "drivers/flash/Kconfig.gecko"
 
-source "drivers/flash/Kconfig.native_posix"
```

Comment:
You forgot to remove this kconfig file itself?

Reply:
removed